jupyterhub/tests/test_db.py

```diff
@@ -35,7 +35,9 @@ def generate_old_db(env_dir, hub_version, db_url):
     check_call([env_py, populate_db, db_url])
 
 
-@pytest.mark.parametrize('hub_version', ['1.0.0', "1.2.2", "1.3.0", "1.5.0", "2.1.1"])
+# changes to this version list must also be reflected
+# in ci/init-db.sh
+@pytest.mark.parametrize('hub_version', ["1.1.0", "1.2.2", "1.3.0", "1.5.0", "2.1.1"])
```

Comment:
Had to bump the oldest version to test from because 1.0 can't create a db with recent mysql, due to #2712